_tools/roles/main.go

```diff
@@ -0,0 +1,214 @@
+// The doc command prints the doc comment of a package-level object.
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/ast"
+	"go/importer"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"strings"
+
+	"golang.org/x/tools/go/loader"
+)
+
+const (
+	// UASTPackage  package containing the roles definition.
+	UASTPackage = "gopkg.in/bblfsh/sdk.v1/uast"
+	// RoleType go type name of the Role type
+	RoleType = UASTPackage + ".Role"
+	// GitHubFilePattern route to the annotation.go file at GitHub
+	GitHubFilePattern = "https://github.com/bblfsh/%s-driver/blob/master/driver/normalizer/annotation.go"
+)
+
+var (
+	// OfficialDriver list of official driver maintanained by bblfsh org.
+	OfficialDriver = map[string]string{
+		"python": "github.com/bblfsh/python-driver/driver/normalizer",
+		"java":   "github.com/bblfsh/java-driver/driver/normalizer",
```

Comment:
We could maybe use github's API to find all projects named *-driver inside bblfsh. But that could be another PR.